testing/unit/test_analyse_text.py

```diff
@@ -0,0 +1,60 @@
+from json import load
+import sys
+sys.path.append('../../taketwo-webapi')
+sys.path.append('../util')
+import unittest
+from unittest.mock import patch
+
+from main import analyse_text
+from main import Text
+
+from util.mock_db_util import getRow
+from util.mock_db_util import setupMocks
+
+class TestAnalyseText(unittest.TestCase):
```

Comment:
I'm not super familiar with python test coding standards but having the class name as the method under test and each method inside the class describing the test case seemed reasonable to me